### PR TITLE
Feature/gettingstarted

### PR DESCRIPTION
**Pull request by paselem**
Improve the getting started experience by adding a set of helper scripts that users can run from the CLI. These should quickly generate all of the required resources and secrets needed to run doAzureParallel w/o needing to do a bunch of clicking around in the Portal.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,29 +57,33 @@ To run your R code across a cluster in Azure, we'll need to get keys and account
 ### Setup Azure Account
 First, set up your Azure Account ([Get started for free!](https://azure.microsoft.com/en-us/free/))
```
Comment (brnleehng) [inline README.md]: Can we also set up your Azure account through Azure Cloud Shell? At least there will be a prompt to login and it'll say create a new account? One less site to deal with?

File: README.md [hunk 2]
```diff
@@ -57,29 +57,33 @@ To run your R code across a cluster in Azure, we'll need to get keys and account
...
 
-Once you have an Azure account, you'll need to create the following two services in the Azure portal:
-- Azure Batch Account ([Create an Azure Batch Account in the Portal](https://docs.microsoft.com/en-us/azure/Batch/batch-account-create-portal))
-- Azure Storage Account (this can be created with the Batch Account)
+Once you have an Azure account, you'll need to create the following resources in Azure:
+- Azure Batch Account
```
Comment (wesholley) [inline README.md]: Can we include a link to a "Getting Started" article like this: https://azure.microsoft.com/en-us/free/?ref=microsoft.com&utm_source=microsoft.com&utm_medium=docs&utm_campaign=visualstudio

This is what we link to in the Batch samples (https://docs.microsoft.com/en-us/azure/batch/quick-run-python).

File: README.md [hunk 3]
```diff
@@ -57,29 +57,33 @@ To run your R code across a cluster in Azure, we'll need to get keys and account
 ### Setup Azure Account
 First, set up your Azure Account ([Get started for free!](https://azure.microsoft.com/en-us/free/))
 
-Once you have an Azure account, you'll need to create the following two services in the Azure portal:
-- Azure Batch Account ([Create an Azure Batch Account in the Portal](https://docs.microsoft.com/en-us/azure/Batch/batch-account-create-portal))
-- Azure Storage Account (this can be created with the Batch Account)
+Once you have an Azure account, you'll need to create the following resources in Azure:
```
Comment (wesholley) [inline README.md]: Can we include a link to instructions for creating a free account?  For anyone new to Azure, this is a good, quick resource to show what you need and is what we link to in the Batch quickstarts: https://azure.microsoft.com/en-us/free/?ref=microsoft.com&utm_source=microsoft.com&utm_medium=docs&utm_campaign=visualstudio

File: README.md [hunk 4]
```diff
@@ -57,29 +57,33 @@ To run your R code across a cluster in Azure, we'll need to get keys and account
...
 
-![Azure Batch Acccount in the Portal](./vignettes/doAzureParallel-azurebatch-instructions.PNG "Azure Batch Acccount in the Portal")
+```sh
+# Download the required setup scripts
+wget https://raw.githubusercontent.com/Azure/doAzureParallel/inst/getting-started/manage_account.sh
 
-For your Azure Storage Account, we need to get:
-- Storage Account Name
-- Storage Account Access Key
+# Create resources in West US (westus). You can find a list of locations by running
```
Comment (wesholley) [inline README.md]: I suggest we point them to pick the location nearest them, which they can get by... (next line, 74), but they can use West US as a default.

File: README.md (hunk 1 above)
Comment (paselem) [reply README.md]: Yes, if they do not have an account yet, they will need to create it here.

File: README.md (hunk 3 above)
Comment (paselem) [reply README.md]: Isn't that was is already referenced on line 58?

File: README.md [hunk 5]
```diff
@@ -57,29 +57,33 @@ To run your R code across a cluster in Azure, we'll need to get keys and account
...
+# 'az account list-locations'
+# Make sure to use the 'name' field and not the 'displayName' field
```
[the remark this answers is not in the document]
Comment (paselem) [reply README.md]: Both are supported but updated text to be consistent

File: README.md (hunk 4 above)
Comment (paselem) [reply README.md]: Updated text to make it clearer.

File: README.md (hunk 3 above)
Comment (wesholley) [reply README.md]: I must have missed that...

File: README.md (hunk 4 above)
Comment (wesholley) [reply README.md]: Looks good

**Comment by brnleehng** [comment]
@paselem Close this PR?

**Comment by paselem** [comment]
Closed in favor of #255

**Pull request closed without merging**